### ISSUE TITLE
Using same ratio for both portrait and landscape images

### ISSUE THREAD

**Issue by marko8994** [guoyingtao/Mantis#87]
Is it possible to make custom ratios to be be fixed(not switching width and height) for both landscape and portrait images. We have some specific usage and we want custom ratios to be the same and not change.

Example:

1. Define two custom ratios
config.addCustomRatio(byVerticalWidth: 3, andVerticalHeight: 4)
config.addCustomRatio(byVerticalWidth: 16, andVerticalHeight: 9)
2. Try to crop portrait image, everything will work fine
3. Try to crop landscape image, custom ratios will have flipped value, they will become 4:3 and 9:16. In this step, it would be nice if custom added ratios remain with same values: 3:4 and 16:9

**Comment by guoyingtao** [comment]
@marko8994 Yes, it is possible. I can add a new config setting for it in the next release.

**Comment by guoyingtao** [comment]
@marko8994 I released Mantis 1.4.11 which includes this feature.
I added `fixRatiosShowType` to Mantis.config
```Swift
public var fixRatiosShowType: FixRatiosShowType = .adaptive
```
You can change this value to .horizontal or .vertical to use the same ratios for both portrait and landscape images

**Comment by marko8994** [comment]
@guoyingtao, I see that you implemented this only in case we use RatioPresenter. Is it possible to add optional login for RatioSelector also since we are not using RatioPresenter?

**Comment by guoyingtao** [comment]
@marko8994 I added fixRatiosShowType setting to RatioSelecor in the master branch.

I also moved fixRatiosShowType to Config.cropToolbarConfig
```Swift
        var config = Mantis.Config()
        config.cropToolbarConfig.ratioCandidatesShowType = .alwaysShowRatioList
        config.cropToolbarConfig.fixRatiosShowType = .vetical
```

Can you check if it is what you want? Then I can release a new version of Mantis.

**Comment by marko8994** [comment]
@guoyingtao, Yup, this is what we wanted. Everything works as expected. Thanks a lot.

**Comment by guoyingtao** [comment]
@marko8994 I just released Mantis 1.4.12 which includes this support.
